src-electron/server/ws-server.js

```diff
@@ -145,7 +156,9 @@ function sendWebSocketData(socket, category, payload) {
  * @param {*} msg
  */
 function sendWebSocketMessage(socket, msg) {
-  doSend(socket, msg)
+  return new Promise((resolve, reject) => {
```

Comment:
why is doSend wrapped within a Promise block? doSend() does not intentionally return any value to be consumed later.

Reply:
It needs to return something because the function that calls it is expecting a resolution. I can probably just take this out for now and just go with the page and have the pop up as a later issue.